.changeset/curvy-flowers-dance.md

```diff
@@ -0,0 +1,6 @@
+---
+'@vercel/next': major
```

Comment:
curious why this is a major? afaict this doesn't have any incompatible changes with older versions